README.md

```diff
@@ -5,11 +5,65 @@
+  - [What is the relationship between resilience engineering and DevOps/SRE?](#what-is-the-relationship-between-resilience-engineering-and-devopssre)
+  - [I’m intrigued, what’s next?](#im-intrigued-whats-next)
+  - [What should I do if I have other questions?](#what-should-i-do-if-i-have-other-questions)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 <!-- prettier-ignore-end -->
 
 # resilience-for-software
 
-Introduction to resilience engineering concepts for software engineers
+The intent of this document is to provide a lightweight introduction to and motivate
+further engagement with resilience engineering through a “narrative FAQ”.
+
+## What's the return on investing time in learning about resilience engineering?
+
+If failures of complex systems (e.g., incidents) are substantially impacting the
+sustainability of your systems, the happiness of your engineers, your ability to meet
+business needs, and/or the happiness of your customers
+([framing borrowed from Honeycomb](https://www.honeycomb.io/framework-for-an-observability-maturity-model-using-observability-to-advance-your-engineering-product/)),
+resilience engineering may offer insight into which approaches to address these issues
+will be effective and high leverage.
+
+## Where do I start? What is resilience engineering?
+
+For definitions and to gain a basic familiarity with the domain, we recommend you start
+by reading [How Complex Systems Fail](https://how.complexsystems.fail) by Richard Cook,
+and then watch
+[Resilience Engineering: The What and How](https://www.youtube.com/watch?v=9f4-Z8Tasa8)
+by John Allspaw.
+
+## What is the relationship between resilience engineering and DevOps/SRE?
+
+This is a complex question without a lightweight answer. We recommend reading the
+preface and conclusion chapters (of course, more if you're inspired) of
+[Accelerate](https://itrevolution.com/book/accelerate/) by Nicole Forsgren PhD, Jez
+Humble, and Gene Kim and
+[Sustainable Operations in Complex Systems with Production Excellence](https://www.infoq.com/articles/production-excellence-sustainable-operations-complex-systems/)
+by Liz Fong-Jones, and comparing the perspectives there with those of Cook and Allspaw
+expressed in the resources linked above.
+
+## I’m intrigued, what’s next?
+
+If you enjoy reading academic papers, see Lorin Hochstein’s
+[paper-centric introduction](https://github.com/lorin/resilience-engineering/blob/master/intro.md).
+If you prefer conference talks, John Allspaw has curated a
+[YouTube playlist](https://www.youtube.com/playlist?list=PLb1aZTnPf3-OMChMkrr6WsokRI6LOnuem).
+Nora Jones also runs a resilience engineering focused Slack Community,
+[Learning From Incidents in Software](https://github.com/norajones/LFI-Slack/blob/master/README.md).
+
+## What should I do if I have other questions?
+
+Your options include
+
+1. Open a PR or issue on this repo
```

Comment:
Do we want to use PRs for questions? Should we perhaps make an Issue Template for asking questions (that coaches the types of questions, what sorts of details we'd like, etc)?

Reply:
what do you think is best? You're like a more expert githubber than I am :-D. This is definitely initial effort (if someone opens a PR that is unhelpful, we can just use "resilience" to handle it case-by-case)

Reply:
A Pull Request usually signals a change to something that the initiator would like to fold into the `master` branch. This PR is a good example. I'd suggest questions are less of a permanent change to the repo and more of a temporary need for work, and as such an issue makes more sense for that. (The outcome of working on an issue is often to file a PR!) An issue is still a bit of process overhead, so we may want to consider having people submit questions via email/tweet/slack.

Reply:
👍 I'll remove PR and leave the suggestion of an issue.

We can always add templates later?